src/js/views/eva-variant-view.js

```diff
@@ -378,7 +468,9 @@ EvaVariantView.prototype = {
                             _this.addAssociatedSSID("ss" + ssIDInfo.accession + "_" + ssIDInfo.data.contig,
                             {"ID": "ss" + ssIDInfo.accession,
                             "Study": _this.getProjectAccessionAnchor(ssIDInfo.data.projectAccession),
-                            "Contig": ssIDInfo.data.contig, "Start": ssIDInfo.data.start,
+                            "Contig": ssIDInfo.data.contig,
+                            "Chromosome": _this.get_chromosome_number_for_accession_with_retries(ssIDInfo.data.contig),
```

Comment:
If I'm reading the SAB member comment correctly, they mentioned that using the term "Contig" for the chromosomal accession is not quite correct; however, we're still doing that. One way to fix this would be to fetch the sequence status and name directly from the assembly report. E. g.:
* We have the assembly accession. For it, fetch information from ENA XML API: https://www.ebi.ac.uk/ena/data/view/GCA_000003055.5&display=xml
* From the output we can have the link to the sequence report: ftp://ftp.ebi.ac.uk/pub/databases/ena/assembly/GCA_000/GCA_000003/GCA_000003055.5_sequence_report.txt
* In the sequence report, we can see that `GK000002.2` has sequence name `2` and the `sequence-role` is `assembled-molecule`